Tests/Parser/Client/fixtures/mediaplayer.yml

```diff
@@ -161,3 +161,24 @@
     type: mediaplayer
     name: Foobar2000
     version: "1.3.10"
+
+- 
+  user_agent: Linux UPnP/1.0 Sonos/61.1-83220 (ZPS1)
+  client:
+    type: mediaplayer
+    name: SONOS
+    version: 61.1-83220
+
+- 
+  user_agent: Linux UPnP/1.0 Sonos/57.4-83220 (ZP120)
+  client:
+    type: mediaplayer
+    name: SONOS
+    version: 57.4-83220
```

Comment:
@sanchezzzhak are we matching versions including a `-` anywhere else? 🤔

Reply:
I checked this information, there is not a single test with a version separated by a dash.